gridmenuview/src/androidTest/java/com/bcc/gridmenuview/MenuItemTest.java

```diff
@@ -0,0 +1,42 @@
+package com.bcc.gridmenuview;
+import android.graphics.drawable.Drawable;
+import androidx.test.platform.app.InstrumentationRegistry;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MenuItemTest  {
+    private MenuItem  menuItem;
+
+    @Before
+    public void setUp()  {
+        menuItem = new MenuItem("Satu",InstrumentationRegistry
+                .getInstrumentation()
+                .getContext()
+                .getDrawable(R.drawable.ic_launcher_background));
+
+    }
+
+    @Test
+    public void MenuItemNotNull(){
+        assertNotNull(menuItem);
+    }
+
+    @Test
+    public void MenuGetTitleShouldReturnString() {
+        assertEquals(menuItem.getTitle(),"Satu");
+    }
```

Comment:
Please change the first characters to lower-case ones, the method name should follow the `camelCase` convention.
You're almost there!

Reply:
ups sorry, coz i delete the test word